gradle/libs.versions.toml

```diff
@@ -134,7 +134,7 @@ managed-rxjava1-interop = "0.13.7"
 managed-slf4j = "1.7.36"
 managed-spock = "2.0-groovy-3.0"
 managed-spotbugs = "4.7.0"
-managed-swagger = "2.2.0"
+managed-swagger = "2.2.1"
```

Comment:
@timyates do we have a task to remove this from core?

Reply:
https://github.com/micronaut-projects/micronaut-core/issues/7727